packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx

```diff
@@ -75,43 +76,43 @@ export const SummaryCard: React.FC = () => {
         {loaded && !loadError && (
           <Gallery hasGutter>
             <GalleryItem className="mco-dashboard__contentColumn">
-              <div className="text-muted mco-summary__item-padding">
+              <Text className="text-muted mco-summary__item-padding mco-dashboard__statusText--size">
                 {t('Clusters')}
-              </div>
-              <div className="mco-dashboard__title mco-summary__item-padding mco-dashboard__title--size">
+              </Text>
+              <Text className="mco-summary__item-padding mco-summary__countText--size mco-dashboard__statusText--weight">
                 {summaryMap.clusters.totalCount}
-              </div>
+              </Text>
               <Divider className="mco-summary__divider--width mco-summary__item-padding" />
               <HealthItem
                 title={t('{{ healthy }} healthy', {
                   healthy: healthyClusters,
                 })}
                 state={HealthState.OK}
                 disableDetails={true}
-                className="mco-summary__item-padding"
+                className="text-muted mco-summary__item-padding"
               />
               <HealthItem
                 title={t('{{ issues }} with issues', {
                   issues: summaryMap.clusters.withIssuesCount,
                 })}
                 state={HealthState.ERROR}
                 disableDetails={true}
-                className="mco-summary__item-padding"
+                className="text-muted mco-summary__item-padding"
               />
             </GalleryItem>
             <GalleryItem className="mco-dashboard__contentColumn">
-              <div className="text-muted mco-summary__item-padding">
+              <Text className="text-muted mco-summary__item-padding mco-dashboard__statusText--size">
                 {t('Applications')}
-              </div>
-              <div className="mco-dashboard__title mco-summary__item-padding mco-dashboard__title--size">
+              </Text>
+              <Text className="mco-summary__item-padding mco-summary__countText--size mco-dashboard__statusText--weight">
```

Comment:
```suggestion
              <Text className="mco-summary__item--padding mco-summary__countText--size mco-dashboard__statusText--weight">
```
shouldn't that be a modifier?

Reply:
it  looks it, this class is used in multiple components, not sure all can take this as modified, let me check